INSTALL.md

```diff
@@ -76,7 +76,6 @@ dnf install cmake cfitsio-devel gcc gcc-gfortran pkgconf-pkg-config
 ```
 conda create -n hstcal -c conda-forge cmake compilers cfitsio pkgconfig python
 conda activate hstcal
-export PKG_CONFIG_PATH="$CONDA_PREFIX/lib/pkgconfig:$PKG_CONFIG_PATH"
```

Comment:
On my Mac I have often had to explicitly set this path.   I tried to test  this just now by making a new hstcal conda environment, but my upgraded Mac is missing some software so I am unable to execute the "cmake".  Argh.  Please just leave this in the recipe for now in case users have to do this.

Reply:
This section  is for Linux only. I did not touch `PKG_CONFIG_PATH` instruction for OSX.